DependencyInjection/SmartboxIntegrationFrameworkExtension.php

```diff
@@ -46,6 +47,7 @@ class SmartboxIntegrationFrameworkExtension extends Extension
         'smartesb.amqp.queue_manager',
         'smartesb.consumers.async_queue',
         'smartesb.drivers.queue.amqp',
+        'smartesb.consumers.php_amqp_lib_signal_queue.class'
```

Comment:
The purpose of this array was to remove the definitions of these services from the container. Based on this, all the new services you have implemented (manager, driver and consumers should be here)

Reply:
Ok